funnel/assets/cypress/fixtures/proposal.json

```diff
@@ -9,5 +9,6 @@
   "location": "Bangalore",
   "room": "Taj M G Road – Banquet hall",
   "venue_room": "in Banquet hall Taj M G Road",
-  "time": "9:05"
+  "time": "9:05",
+  "email": "cypress-user@gmail.com"
```

Comment:
is this valid email format?

Reply:
It's valid email but not a valid gmail format. I have changed the domain to random one.